### PR TITLE
Create markup for signupform

### PR DESCRIPTION
**Pull request by Pedrael**

### CODE REVIEW

File: src/components/CustomisedButton.tsx [hunk 1]
```diff
@@ -11,6 +11,11 @@ const CustomisedButton = styled(Button)({
...
   padding: '18px',
   textTransform: 'capitalize',
+  '&.Mui-disabled': {
+    backgroundColor: '#B4B4B4',
+    color: 'white',
+    cursor: 'not-allowed',
+  },
 })
 
 export default CustomisedButton
```
Comment (ReidenXerx) [inline src/components/CustomisedButton.tsx:21]: Not customized but valuable name

File: src/components/CustomisedImageUpload.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  onImageLoad: (imageData: ImageData) => void
+}
+
+export function CustomisedImageUpload({
+  onImageLoad,
+}: CustomisedImageUploadProps) {
+  const [selectedFile, setSelectedFile] = useState<File | null>(null)
+  const [isSelected, setSelected] = useState<boolean>(false)
+
+  const handleFileChange = (event: ChangeEvent<HTMLInputElement>) => {
```
Comment (ReidenXerx) [inline src/components/CustomisedImageUpload.tsx]: Destructurization

File: src/components/CustomisedTextField.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      borderWidth: color === 'error' ? ' 2px' : undefined,
+    },
+    '&.Mui-focused fieldset': {
+      borderWidth: color === 'primary' ? '1px' : undefined,
+      borderColor: color === 'primary' ? textFieldBorder : undefined, // Border color when focused
+    },
+  },
+}))
+
+export default CustomisedTextField
```
Comment (ReidenXerx) [inline src/components/CustomisedTextField.tsx:57]: Naming

File: src/components/CustomisedImageUpload.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  name: string
+  size: number
+  type: string
+  url: string
+}
+interface CustomisedImageUploadProps {
+  onImageLoad: (imageData: ImageData) => void
+}
+
+export function CustomisedImageUpload({
```
Comment (ReidenXerx) [inline src/components/CustomisedImageUpload.tsx]: Why its function but not classic const?

**Review by ReidenXerx — CHANGES_REQUESTED** [review]

File: src/components/CustomisedButton.tsx (hunk 1 above)
Comment (Pedrael) [reply src/components/CustomisedButton.tsx:21]: will be fixed in the next pr

File: src/components/CustomisedTextField.tsx (hunk 3 above)
Comment (Pedrael) [reply src/components/CustomisedTextField.tsx:57]: will be fixed in the next pr

File: src/components/CustomisedImageUpload.tsx (hunk 2 above)
Comment (Pedrael) [reply src/components/CustomisedImageUpload.tsx]: fixed

File: src/components/CustomisedImageUpload.tsx (hunk 4 above)
Comment (Pedrael) [reply src/components/CustomisedImageUpload.tsx]: fixed

**Review by ReidenXerx — APPROVED** [review]

**Pull request merged**